benchmarks/torchbench.py

```diff
@@ -194,6 +194,16 @@
 }
 
 
+SKIP_ACCURACY_CHECK_MODELS = {
+    # Models too large to have eager, dynamo and fp64_numbers simultaneosuly
```

Comment:
In this case,  can we write the results to CPU and then compare?

Reply:
Hmm, I noticed the `_large` in their names, maybe we are ok here.

Reply:
I think we should use CPU for comparison in general. It might remove some OOM error for CI. Lets handle that in a separate PR.